### PR TITLE
GUI with newer Qt version for MacOS 10.12+

### PR DESCRIPTION
**Pull request by g-maxime**
Signed-off-by: Maxime Gervais <gervais.maxime@gmail.com>

### CODE REVIEW

File: src/QCToolsBundle/Resources/views/Download/macTable.html.twig [hunk 1]
```diff
@@ -10,9 +10,14 @@
     <td colspan="5">&nbsp;<br>&nbsp;</td>
 </tr>
 <tr>
-    <th rowspan="2"><br>10.10&nbsp;(Yosemite),<br>10.11&nbsp;(El&nbsp;Captain),<br>10.12&nbsp;(Sierra),<br>10.13&nbsp;(High&nbsp;Sierra),<br>10.14&nbsp;(Mojave)<br>10.15&nbsp;(Catalina)</th>
-    <th rowspan="2">x86_64</th>
-    <td><abbr title="Graphical User Interface">GUI</abbr></td>
+    <th rowspan="3"><br>10.10&nbsp;(Yosemite),<br>10.11&nbsp;(El&nbsp;Captain),<br>10.12&nbsp;(Sierra),<br>10.13&nbsp;(High&nbsp;Sierra),<br>10.14&nbsp;(Mojave)<br>10.15&nbsp;(Catalina)</th>
```
Comment (JeromeMartinez) [inline src/QCToolsBundle/Resources/views/Download/macTable.html.twig]: 10.10 and 10.11 not compatible.
Also in the remarks part (right column), something like "Include support of dark theme on macOS 10.14 (Mojave) onward".

File: src/QCToolsBundle/Resources/views/Download/macTable.html.twig [hunk 2]
```diff
@@ -10,9 +10,14 @@
...
+    <th rowspan="3">x86_64</th>
+    <td><abbr title="Graphical User Interface">GUI</abbr> (mac0S 10.12 and newer) </td>
+    <td><a href="//mediaarea.net/download/binary/qctools/1.1/QCTools_1.1_mac_1012.dmg">v1.1</a></td>
+    <td>&nbsp;</td>
+</tr>
+<tr>
+    <td><abbr title="Graphical User Interface">GUI</abbr> (mac0S 10.10 and 10.11) </td>
```
Comment (JeromeMartinez) [inline src/QCToolsBundle/Resources/views/Download/macTable.html.twig]: As it works with newer versions, I would list all of them (10.10 to 10.15).

**Review by JeromeMartinez — CHANGES_REQUESTED** [review]

File: src/QCToolsBundle/Resources/views/Download/macTable.html.twig (hunk 2 above)
Comment (g-maxime) [reply src/QCToolsBundle/Resources/views/Download/macTable.html.twig]: I have reworked my PR

File: src/QCToolsBundle/Resources/views/Download/macTable.html.twig [hunk 3]
```diff
@@ -9,6 +9,13 @@
 <tr>
     <td colspan="5">&nbsp;<br>&nbsp;</td>
 </tr>
+<tr>
+    <th><br><br>10.12&nbsp;(Sierra),<br>10.13&nbsp;(High&nbsp;Sierra),<br>10.14&nbsp;(Mojave)<br>10.15&nbsp;(Catalina)</th>
```
Comment (retokromer) [inline src/QCToolsBundle/Resources/views/Download/macTable.html.twig]: Nit: I would suggest either to add or to remove the comma before `<br>`.

**Review by JeromeMartinez — APPROVED** [review]

**Pull request merged**